openwpm/task_manager.py

```diff
@@ -620,16 +603,17 @@ def execute_command_sequence(
         int  -> index of browser to send command to
         """
 
-        # Block if the aggregator queue is too large
-        agg_queue_size = self.storage_controler_handle.get_most_recent_status()
-        if agg_queue_size >= AGGREGATOR_QUEUE_LIMIT:
-            while agg_queue_size >= AGGREGATOR_QUEUE_LIMIT:
+        # Block if the storage controller has too many unfinished records
+        # TODO create tests that these numbers are still meaningful
```

Comment:
I suggest we remove this comment and create an issue. Otherwise it will be lost.

Reply:
I filed https://github.com/mozilla/OpenWPM/issues/861 and removed the comment.